modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -355,7 +363,7 @@ impl LightningClientModule {
         self.send_internal(
             gateway_api,
             invoice,
-            PaymentFee::TEN_PROMILLE_PLUS_50_SATS,
+            PaymentFee::SEND_FEE_LIMIT_DEFAULT,
             EXPIRATION_DELTA_LIMIT_DEFAULT,
         )
         .await
```

Comment:
This is the important change here, the client accepts a higher fee to enable the gateway to route with higher fees.